### ISSUE TITLE
Bump package.json version

### ISSUE THREAD

**Issue by rarkins** [renovateapp/renovate#861]
As suggested by @gunar [on twitter](https://twitter.com/gunar/status/913869355275997184), this would see Renovate bump the version in `package.json`.

**Comment by rarkins** [comment]
I suggest we'd call this "bumpVersion" and make it boolean (false by default). We would do this manually rather than actually running "npm version patch".

**Comment by rarkins** [comment]
Perhaps "bumpVersion" and default to "none" but allow "patch" or "minor" as values.

**Comment by rarkins** [comment]
Waiting on #1016, otherwise it's hard to make sure we don't accidentally bump it twice if branch is updated with second commit.

**Comment by gunar** [comment]
@rarkins think you can move this forward now? :)

**Comment by rarkins** [comment]
@gunar yes, good idea. Nowadays, Renovate always has only one commit per branch, so it makes it much easier.

**Comment by rarkins** [comment]
@gunar now implemented and live in the app, if you'd like to try it out again.

e.g. to bump version *every time*, just add this to the root of your `renovate.json`:
```
  "bumpVersion": true
```

If you only want it for `dependencies` (not `devDependencies`), then add this:
```
  "dependencies": {
    "bumpVersion": true
  }
```

**Comment by gunar** [comment]
@rarkins you are awesome! will try asap.

**Comment by NearAutomata** [comment]
@rarkins I'd like to quickly ask about `bumpVersion`. It seems like the setting `"bumpVersion": "patch"` is ignored if the major version is 0, but works for numbers greater than 0. Is this an intended behaviour?

**Comment by rarkins** [comment]
Not intended. Could you raise a bug report?

**Comment by NearAutomata** [comment]
@rarkins Apparently the issue resolved all of a sudden, but I stumbled upon another issue I will raise a bug report for when I am able to recreate it.
